libshviotqt/src/rpc/rpc.cpp

```diff
@@ -17,7 +17,7 @@ void Rpc::registerMetaTypes()
 	}
 }
 
-shv::chainpack::RpcValue::DateTime Rpc::toRpcDateTime(const QDateTime &d)
+chainpack::RpcValue Rpc::toRpcDateTime(const QDateTime &d)
```

Comment:
to jsi naimplementoval `toRpcValue()` to je uplne jina metoda, k cemu to potrebujes?

Reply:
Ono to takhle bylo historicky, jeste z doby kdy existovalo RpcValue::DateTime null. Opravil jsem to aby tam zustala jen funkce toRpcValue.